lxd/network/network_utils.go

```diff
@@ -279,6 +279,19 @@ func GetDevMTU(devName string) (uint32, error) {
 	return uint32(mtu), nil
 }
 
+// GetDevTXqlen retrieves the current txqlen setting for a named network device.
+func GetDevTXqlen(devName string) (string, error) {
```

Comment:
This should be `GetDevTXQlen` (upper case Q) to be consistent with `SetTXQlen` above.

Reply:
This function should return an integer argument.

Reply:
Can we call this `GetTXQueueLength` for clarity please.